### DISCUSSION TITLE
can support multi References?

### DISCUSSION THREAD

**Discussion by chanox** [ikvmnet/ikvm#357] [Q&A]
<ItemGroup>
   <IkvmReference Include="helloworld.jar"/>
   <IkvmReference Include="helloworld_pro.jar"/>

   <IkvmReference Include="helloworld-2.jar">
      <References>helloworld.jar</References>
      <References>helloworld_pro.jar</References>
   </IkvmReference>
</ItemGroup>

**Comment by wasabii** [comment]
I don't understand the question.

**Reply by chanox** [reply]
sory not show code.. like this one jar file but 2 reference file

ItemGroup
   IkvmReference Include="helloworld.jar"
   IkvmReference Include="helloworld_pro.jar"

   IkvmReference Include="helloworld-2.jar"
      References helloworld.jar References
      References helloworld_pro.jar References
   kvmReference
ItemGroup

**Reply by wasabii** [reply]
I don't think the names of two jar files is going to help.

**Reply by chanox** [reply]
in my case, i use 2 jar reference file commons-cli-1.4.jar and javassist-3.23.1-GA.jar
is that posible?

**Reply by wasabii** [reply]
> References | Optional *semi-colon separated list* of other IkvmReference identity values to specify as a reference to the current one. For example, if foo.jar depends on bar.jar, include both as IkvmReference items, but specify the identity of bar.jar on the References metadata of foo.jar.

**Reply by chanox** [reply]
thanks it works... i change reference like this.. commons-cli-1.4.jar;javassist-3.23.1-GA.jar
